src/Subscriptions/resources/admin/components/SubscriptionDetailsPage/Tabs/Overview/SubscriptionStats/index.tsx

```diff
@@ -0,0 +1,47 @@
+import StatWidget from '@givewp/admin/components/StatWidget';
+import {__} from '@wordpress/i18n';
+import {amountFormatter} from '@givewp/admin/utils';
+import { Donation } from '@givewp/donations/admin/components/types';
+import styles from './styles.module.scss';
+
+/**
+ * @unreleased
+ */
+type SubscriptionStatsProps = {
+    donations: Donation[];
+    currency: string;
+    totalInstallments: number;
+    loading: boolean;
+}
+
+/**
+ * @unreleased
+ */
+export default function SubscriptionStats({donations, currency, totalInstallments, loading}: SubscriptionStatsProps) {
+    const ongoingInstallments = totalInstallments === 0;
+    const paymentsCompleted = donations?.length;
```

Comment:
Do we need to check for the donation status or is that already handled before this?